src/core/mount.c

```diff
@@ -1104,9 +1104,15 @@ static int mount_apply_graceful_options(Mount *m, const MountParameters *p, char
                         return r;
 
                 r = mount_option_supported(p->fstype, k ?: *o, v);
+                if (r == -EAGAIN) {
+                        log_unit_warning_errno(UNIT(m), r,
+                                               "x-systemd.graceful-option= used but not supported by file system %s, suppressing all.",
```

Comment:
Maybe better to explicitly say what is unsupported.
Also, maybe `by file system` -> `by the kernel` ?

Reply:
by file system is accurate. We don't support kernels without fsopen anymore after all, and several file systems (such as btrfs) got ported over only recently